changelog/db.changelog-1.5.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+ <changeSet author="Abidemi Adio" id="1">
+    <tagDatabase tag="v1.4.0" />
+  </changeSet>
+  <changeSet author="Abidemi Adio" id="2">
+    <addColumn tableName="calculations">
+      <column name="invoiceNumber" type="VARCHAR(20)" />
```

Comment:
This is `varchar(30)` this is the enriched `invoiceNumber`

Reply:
NOPE! I'm wrong. 

This is correct, could you rectify this in ERD(it says `varchar(30)`